src/page/EditProbe/EditProbe.test.tsx

```diff
@@ -79,6 +80,78 @@ describe(`Private probes`, () => {
     const tokenValue = await screen.findByText(UPDATED_PROBE_TOKEN_RESPONSE);
     expect(tokenValue).toBeInTheDocument();
   });
+
+  it('probe status updates automatically', async () => {
+    jest.useFakeTimers();
+    server.use(
+      apiRoute('listProbes', {
+        result: () => {
+          return {
+            json: [OFFLINE_PROBE],
+          };
+        },
+      })
+    );
+
+    renderEditProbe(OFFLINE_PROBE);
+
+    // Initial render - should call once
```

Comment:
instead of the comment, don't you want to add an assertion making sure it's just called 1 times?